### ISSUE TITLE
Progress circle should use fixed-width digit font

### ISSUE THREAD

**Issue by rillig** [docker/kitematic#3658]
### Expected behavior

During "Starting Docker VM", the progress is visualized in a circle.

![image](https://user-images.githubusercontent.com/3233724/37532565-a5225bb2-293f-11e8-81a9-c7a19f9a26d7.png)

When the percentage changes from 60 to 61, the percentage character should stay in place, to avoid any flickering appearance. Same for the 6, which also didn't change.

### Actual behavior

The whole string `%d%%` is centered, therefore the `6` and the `%` each jump by a few pixels. This makes the screen more nervous than necessary.

### Workaround (for developers)

If using a font with fixed-width digits is not possible, it would help to draw each digit individually in this case, reserving the maximum necessary width for each digit.

**Comment by FrenchBen** [comment]
@rillig Thanks for the issue and your attention to detail - Is this something you'd consider opening a PR for?

**Comment by rillig** [comment]
Yes, I definitely and intentionally considered opening a PR for this.

Kitematic has a minimalistic UI, in which each non-white pixel has its purpose. Therefore, when some of these pixels flicker, they make me look at them, only to discover that nothing essential has changed. Whether the progress is at 60% or at 61% doesn't really matter, nevertheless some more pixels changed their color.

This contradicts the quiet and unobtrusive style and design that Kitematic is using.

**Comment by shandheap** [comment]
Hey @rillig, are you still working on this? If not, I'd like to work on this issue for my first PR.

**Comment by rillig** [comment]
Hi @shandheap, I have never "worked" on this. I just reported it in the hope that it would be fixed someday.
